cluster/config.go

```diff
@@ -109,6 +109,11 @@ func BuildContainerConfig(c dockerclient.ContainerConfig) *ContainerConfig {
 	// remove affinities/constraints/reschedule policies from env
 	c.Env = env
 
+	if parts := strings.SplitN(c.HostConfig.NetworkMode, "/", 2); len(parts) == 2 {
```

Comment:
@mavenugo Is there any requirement to support `/` in network name? Can we update networking to not allow `/`? I feel like we might have other problems if we allow arbitrary characters.